execution_workflows/LABRAT/modules/salmon_quant.nf

```diff
@@ -7,7 +7,9 @@ def options    = initOptions(params.options)
 process SALMON_QUANT {
         tag "$sample"
         publishDir "${params.outdir}/labrat/salmon/", mode: params.publish_dir_mode
-        container "quay.io/biocontainers/salmon:0.14.0--h86b0361_1"
+        container "quay.io/biocontainers/salmon:1.6.0--h84f40af_0"
```

Comment:
`LABRAT` mentions it depends on [`salmon 0.14.0`](https://github.com/TaliaferroLab/LABRAT#option-2-manual-installation). I personally think it is good to update `salmon` to the most recent version given that everything runs fine downstream

Reply:
At least for the samples I was using it was fine. And LABRAT uses the salmon output, this should not change much. 
However, more testing might be needed to double-check.